examples/compute/instances.go

```diff
@@ -15,16 +14,10 @@ import (
 func main() {
 	keyID := os.Getenv("SDC_KEY_ID")
 	accountName := os.Getenv("SDC_ACCOUNT")
-	keyPath := os.Getenv("SDC_KEY_FILE")
```

Comment:
👍 

I'm totally on-board with removing `SDC_KEY_FILE` as it's not idiomatic. It was simply an environment variable I was using to manually force the path for `authentication.NewPrivateKeySigner`. `authentication.NewSSHAgentSigner` is much better here since its less config.